code/datums/mutantraces.dm

```diff
@@ -836,6 +836,38 @@
 			mob.base_body_temp = initial(mob.base_body_temp)
 		. = ..()
 
+	emote(var/act, var/voluntary)
+		switch(act)
+			if ("whip")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> whips their tail", "<i>whips their tail</i>")
```

Comment:
Also, for most of these, I'd recommend reducing or removing the cooldown time on any of the non-noisemaking, non-animated emotes, since they're mostly just text and there should be no harm in spamming them. And also use the time unit defines, like instead of `50`, you'd want to use `5 SECONDS`, since the timing is in deciseconds.